File: ruby/001.rb
def sumMultiplesOfThreeAndFiveBelow (n)
  sum = 0

  for i in 0..(n-1)
    if i % 3 == 0 or i % 5 == 0
      sum += i
    end
  end

  return sum
end

answer = sumMultiplesOfThreeAndFiveBelow 1000
puts answer

File: javascript/001.js
function sumMultiplesOfThreeAndFiveBelow(n) {
  var sum = 0;

  for (var i = 0; i < n; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
      sum += i;
    }
  }

  return sum;
}

var answer = sumMultiplesOfThreeAndFiveBelow(1000);
console.log(answer);